package.json

```diff
@@ -1,81 +1,94 @@
-    "doc": "./node_modules/esdoc/out/src/ESDocCLI.js"
-  },
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "babel-core": "^6.14.0",
-    "babel-eslint": "^6.1.2",
-    "babel-loader": "^6.2.5",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-regenerator": "^6.16.1",
-    "babel-preset-es2015": "^6.14.0",
-    "chai": "^3.5.0",
-    "esdoc": "^1.0.3",
-    "esdoc-inject-script-plugin": "^1.0.0",
-    "esdoc-standard-plugin": "^1.0.0",
-    "eslint": "^3.6.1",
-    "eslint-config-google": "^0.6.0",
-    "fs-extra": "^4.0.0",
-    "karma": "^1.3.0",
-    "karma-babel-preprocessor": "^6.0.1",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-edge-launcher": "^0.4.2",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-ie-launcher": "^1.0.0",
-    "karma-mocha": "^1.1.1",
-    "karma-mocha-reporter": "^2.2.0",
-    "karma-safari-launcher": "^1.0.0",
-    "karma-webpack": "^1.8.0",
-    "lodash": "^4.16.2",
-    "mocha": "^3.0.2",
-    "mocha-bamboo-reporter": "^1.1.1",
-    "mocha-webpack": "^0.6.0",
-    "npm-cli-login": "0.0.10",
-    "sinon": "^1.17.6",
-    "webpack": "^1.13.2"
-  },
-  "dependencies": {
-    "axios": "^0.14.0",
-    "babel-regenerator-runtime": "^6.5.0",
-    "base64-js": "^1.2.0",
-    "base64url": "^2.0.0",
-    "bs58": "^4.0.1",
-    "buffer": "3.4.3",
-    "cross-env": "^5.2.0",
-    "es6-promise": "^3.3.1",
-    "fast-sha256": "^1.0.0",
-    "json-loader": "^0.5.4",
-    "json-stable-stringify": "^1.0.1",
-    "tweetnacl": "^1.0.0"
-  }
+	"name": "token-io",
+	"version": "2.0.0",
+	"description": "Token JavaScript SDK",
+	"license": "ISC",
+	"author": {
+		"name": "Token",
+		"url": "https://token.io"
+	},
+	"homepage": "https://github.com/tokenio/sdk-js",
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/tokenio/sdk-js.git"
+	},
+	"scripts": {
+		"type": "flow check",
+		"lint": "eslint .",
+		"test": "flow check && eslint test && mocha-webpack --webpack-config test/webpack.config.js --full-trace --check-leaks -t 60000 -c \"test/**/*.spec.js\" && rimraf keys",
+		"testBrowser": "flow check && eslint test && TARGET=browser karma start test/karma.config.js",
+		"bambooDev": "flow check && npm run lint && mocha-webpack --webpack-config test/webpack.config.js -t 60000 -c -R mocha-bamboo-reporter  \"test/**/*.spec.js\"",
+		"build:cjs": "rollup -c",
+		"build:esm": "cross-env FORMAT=esm rollup -c",
+		"build:cjs:browser": "cross-env TARGET=browser rollup -c",
+		"build:esm:browser": "cross-env FORMAT=esm TARGET=browser rollup -c",
+		"build:iife": "cross-env FORMAT=iife TARGET=browser rollup -c",
+		"build": "flow check && eslint src && npm run build:cjs && npm run build:esm && npm run build:cjs:browser && npm run build:esm:browser && npm run build:iife",
+		"proto": "node proto.js",
```

Comment:
can we force version to be passed in and specify proto version in `package.json`

http://www.marcusoft.net/2015/08/npm-scripting-configs-and-arguments.html#npm-configuration

Reply:
Would be nice if we could run it on each build (and don't update if version matches), but that can be an improvement for later

Reply:
this command should usually be run before building right? here's what i was thinking
1. lib-proto updates, so we need to update sdk-js
2. we run yarn proto to fetch the updates, then develop new changes 
3. build

Reply:
Yeah but if `yarn proto` was built into the `build`, then we can just update the version and build without needing to execute a separate command